helpers/util.sh

```diff
@@ -318,12 +318,11 @@ buildmw() {
 }
 
 build() {
-    SPECS=$@
-    if [ -z "$SPECS" ]; then
+    if [ $# -eq 0 ] || [ -z "$1" ]; then
         minfo "No spec file for package building specified, building all I can find."
-        SPECS="rpm/*.spec"
+        set -- "rpm/*.spec"
     fi
-    for SPEC in $SPECS ; do
+    for SPEC in "$@" ; do
```

Comment:
this breaks when multiple specs are passed in cmdline e.g.
```
rpm/dhd/helpers/build_packages.sh \
 --mw=https://github.com/mer-hybris/droid-system-sony-template --do-not-install
 --spec=rpm/droid-system-$HABUILD_DEVICE.spec
 --spec=rpm/droid-system-$HABUILD_DEVICE-$DEVICE.spec
 ```
mb2 becomes
`mb2 -s  rpm/droid-system-pdx213.spec  rpm/droid-system-pdx213-xqbt52.spec  -t sony-xqbt52-aarch64`...

Reply:
that's because build() was called with quotes around the passed spec files in buildmw I changed.

Reply:
it works now thanks!